tasks/main.yml

```diff
@@ -20,23 +14,91 @@
     __ssh_vars_file: "{{ role_path }}/vars/{{ item }}"
   when: __ssh_vars_file is file
 
-# Examples of some tasks:
 - name: Ensure required packages are installed
   package:
-    name: "{{ __ssh_packages }}"
+    name: "{{ __ssh_packages + ssh_additional_packages }}"
     state: present
 
-- name: Ensure required services are enabled and started
-  service:
-    name: "{{ item }}"
-    state: started
-    enabled: yes
-  loop: "{{ __ssh_services }}"
+- debug:
+    var: "{{ ssh_user }}"
+
+- name: Gather information about the user for user configuration
+  getent:
+    database: passwd
+    key: "{{ ssh_user }}"
+  when:
+    - ssh_user is not none
```

Comment:
do you need to `register` the result of this task, or does this module automatically set `getent_passwd` as a global variable?

Reply:
https://docs.ansible.com/ansible/latest/collections/ansible/builtin/getent_module.html -- `getent` automatically populates the variable `getent_passwd`. It was confusing for me too, but it looks like it is how it works.